### ISSUE TITLE
[$10][MSFT-46] [Login] "Show" button foreground and background colors won't meet WCAG 2 AA contrast ratio thresholds

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2767]
**Steps to Reproduce**
1. Go to Go to https://accounts.topcoder.com/member
2. Check the Show button

**Expected Result**
Expected contrast ratio of 4.5:1

- Passes AA, if you change background color to #757584. New contrast ratio: 4.53
- Passes AA, if you change text color to #616161. New contrast ratio: 4.53

Some people with low vision experience low contrast, meaning that there aren't very many bright or dark areas. Everything tends to appear about the same brightness, which makes it hard to distinguish outlines, borders, edges, and details. Text that is too close in luminance (brightness) to the background can be hard to read.

**Actual Result**
Element has insufficient color contrast of 1.36 (foreground color: #ffffff, background color: #dcdce0, font size: 9.0pt, font weight: normal).

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: AA
**WCAG Success Criterion**: 1.4.3 Contrast (minimum)
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=58da9b68%2Df2a3%2D45d2%2Dbbae%2D15abdfd0865b&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=true&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F25

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096300 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096300 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096300 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@veshu PR https://github.com/appirio-tech/accounts-app/pull/225

**Comment by mishacucicea** [comment]
@Dara-K is the #757584 suggested color ok here?

**Comment by tosha5252** [comment]
Button color is now #59a7ff...which still fails WCAG standards...please change to the new blue color #006AD7

**Comment by Dara-K** [comment]
#006AD7 is the new blue color for the button - that is correct

**Comment by mishacucicea** [comment]
File already changed in that location, reverting won't have any effect.
Opening for pickup.

**Comment by PrakashDurlabhji** [comment]
@mishacucicea i did not understand what u mean? i can complete this in some time, please do not unassign me.

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096300 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by mishacucicea** [comment]
@PrakashDurlabhji - you were inactive, so I had to do this, sorry.

**Comment by PrakashDurlabhji** [comment]
@mishacucicea sorry for delay.
@veshu @tosha5252 new PR https://github.com/appirio-tech/accounts-app/pull/261

**Comment by mishacucicea** [comment]
@PrakashDurlabhji Approved, made some changes myself to avoid too many feedback loops (next time will reject).
Please don't treat the items superficially, reuse existing colors  (define them in the tc-ui-variables.scss if they are not there), if something doesn't look good, ask don't just try to minimally satisfy the requirements.

@tosha5252 , @Dara-K The button text color was changed to white to be consistent with the LOG IN button. Note that the color for LOG IN is a bit different...

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097511 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097511 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by drasticdpk** [comment]
No updated suggested color, please update accordingly. 

![image](https://user-images.githubusercontent.com/25297859/61950443-8a012c80-afcd-11e9-9b4b-7f52000a520a.png)

**Comment by PrakashDurlabhji** [comment]
@mishacucicea it has become really difficult to cope up with colors because in mostly all mine tickets I had to update color twice hence its all confusion. I will work on it again, and as all reviewers are not adding feedback label on ticket hence it becomes more difficult. rather than marking code review fail on PR, also feedback label should be added on original ticket then it becomes smooth to track it and update immediately

**Comment by Dara-K** [comment]
@mishacucicea  @PrakashDurlabhji  Please update both buttons to the same blue color: 006AD7.

We are redesigning the Login and Registration page as we speak, so this page will be updated soon.

Thank you.

**Comment by mishacucicea** [comment]
@drasticdpk This issue talks about Show button, not Login, so please don't fail the issue if it's respecting the initial requirements.

We can treat login in another issue (although it might be fixed already in other branches), let's close this one, too much overhead for a 1 line change.

**Comment by tosha5252** [comment]
Button color showing still not changed....it is still #59a7ff...please change to the new blue color #006AD7
Env:
Windows 10
Browser: Chrome

**Comment by mishacucicea** [comment]
@tosha5252 The Show button is fixed:
![image](https://user-images.githubusercontent.com/1721199/62006939-2a7f5a00-b150-11e9-91db-b426f717ead7.png)

**Comment by PrakashDurlabhji** [comment]
@tosha5252 @drasticdpk @sushilshinde  can we close this?

**Comment by mishacucicea** [comment]
@tosha5252 Please try again in incognito window.

**Comment by nithyaasworld** [comment]
@mishacucicea I tried in incognito and I still do not see the expected colour. Am I missing something?

![image](https://user-images.githubusercontent.com/5114190/62091396-4f79e700-b28e-11e9-805a-c8aaf1d456eb.png)

**Comment by nithyaasworld** [comment]
@mishacucicea I tested this on Windows Chrome. Can you confirm that you checked it on Windows Chrome too? Thanks.

**Comment by ajefts** [comment]
I tested this on Mac Chrome and I'm not seeing the update.  Here is what I see:

![image](https://user-images.githubusercontent.com/3812270/62093403-2b46e780-b247-11e9-8263-f2a49bae1cb3.png)

**Comment by nithyaasworld** [comment]
Hi @ajefts Can you please enter some value for the input fields and then check the Show button Colour? Thank You.

**Comment by ajefts** [comment]
@nithyaasworld When I enter values or click on it the contrast is ok.

**Comment by nkumar-topcoder** [comment]
@nithyaasworld could be cache issue, refreshed cloudfront. pls check

**Comment by nithyaasworld** [comment]
@nkumar-topcoder Can you please deploy this fix again to dev environment? Seems like its not available.

**Comment by nithyaasworld** [comment]
Show button when not in focused (Gray colour) has right contrast. Whereas, when its focused (blue button), its not of the expected contrast.

**Comment by nkumar-topcoder** [comment]
@nithyaasworld @ajefts can I push to prod as verified. Or is contrast still an issue ?

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097511 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by r0hit-gupta** [comment]
PR at https://github.com/appirio-tech/accounts-app/pull/286

**Comment by r0hit-gupta** [comment]
@nithyaasworld @nkumar-topcoder @mishacucicea please review.

**Comment by PrakashDurlabhji** [comment]
@r0hit-gupta it is with me, i am on it , why did u unassign me?

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097511 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by r0hit-gupta** [comment]
@sushilshinde please confirm.

**Comment by PrakashDurlabhji** [comment]
@sushilshinde i am waiting for above discussion to end due to deployment issue and its not open for pickup and started taking issues, i need to see whether my other issues are also taken, i just came online

**Comment by lakshmiathreya** [comment]
As discussed in the review call this evening, there is consensus that we can put this issue On Hold - take it off the Prod merge list and resume review of this later. Updating Label. @drasticdpk @tosha5252 fyi

**Comment by mtwomey** [comment]
I made this color change directly (to #006AD7). PR appirio-tech/accounts-app#286 is referencing $tc-dark-blue-110 which is still not the recommended color.

**Comment by lakshmiathreya** [comment]
Verified on Chrome - Show button has Designer's choice blue - #006AD7. Closing Issue.

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097511<br/><br/>```This is an automated message for crazyk via Topcoder X```
